NSS-Day-Cohort-66/rare-api-dapper-pirates
# added tests and category

This is the first of many tests in our application. added packages 

_In the project directory only_
- **tests**
- **__init__.py**
- **category_tests.py**
- **comment_tests.py**
- **post_tests.py**

_In the rareapi directory /fixtures directory_

- **token.json**
- **user.json**

_In the rareapi views directory_

- **posts.py**

**TO TEST**
In your rare-api-dapper-pirates
1. `git checkout main`
2. `git fetch`
3. `git checkout DL `
4. `code . `
5. (optional)  be sure your virtual environment is running with `pipenv shell` and an interpretor is selected.

- Make sure the added packages are present and then start your debugger to make sure there are no issues or exceptions.
- go back to your terminal and run the following command to test the PostTest Test `python3 manage.py test tests -v 1`
- verify you get the following response: 

` Found 1 test(s).
Creating test database for alias default 
System check identified no issues (0 silenced).

Ran 1 test in 0.039s

OK
Destroying test database for alias`
